src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java

```diff
@@ -53,6 +53,8 @@ public class ChatListener {
 	private final NotEnoughUpdates neu;
 	private static final Pattern SLAYER_XP = Pattern.compile(
 		"   (Spider|Zombie|Wolf|Enderman|Blaze) Slayer LVL (\\d) - (?:Next LVL in ([\\d,]+) XP!|LVL MAXED OUT!)");
+
+	private static final Pattern SKYBLOCK_LVL_MESSAGE = Pattern.compile("\\[(\\d{1,3})\\] .*");
```

Comment:
not future proof. what if someone gets sb level 1000

Reply:
/togglechat :+1: